hudi-common/src/main/java/org/apache/hudi/metadata/HoodieMetadataFileSystemView.java

```diff
@@ -73,4 +73,16 @@ public void close() {
       throw new HoodieException("Error closing metadata file system view.", e);
     }
   }
+
+  @Override
+  public void sync() {
```

Comment:
IIUC this is to handle the file system view refreshes

Reply:
Yes